mypy.ini

```diff
@@ -1,5 +1,12 @@
 [mypy]
 ignore_missing_imports = True
+exclude = (?x) (
```

Comment:
Seems like a workaround to something. Why was this added?

Reply:
Was added since we had mypy complain on a lot of files that aren't supported anymore. I did not want to mess around with deleting them and do even more stuff in this PR. I can do a follow-up PR and try to revert these back, if that's OK with you?

Reply:
> I can do a follow-up PR and try to revert these back, if that's OK with you?

Sounds good!

Reply:
Added #6507 to ensure we remember.